content/python/concepts/time-module/terms/time-ctime/time-ctime.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'time.ctime()'
```

Comment:
```suggestion
Title: '.ctime()'
```

Reply:
@Dusch4593 Do we have a reference document that includes these details. I modeled this entry on the py math module which uses the format I've applied here. I feel like I'm always referencing formatting that isn't considered the "current" standard.